rc/filetype/org.kak

```diff
@@ -0,0 +1,262 @@
+# https://orgmode.org/ - your life in plain text
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.]org %{
+    set-option buffer filetype org
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=org %{
+    require-module orgmode
+    hook window InsertChar \n -group org-indent org-indent-on-new-line
+    hook -once -always window WinSetOption filetype=.* %{ remove-hooks window org-.+ }
+    org-parse-file
+}
+
+hook -group org-highlight global WinSetOption filetype=org %{
+    add-highlighter window/org ref org
+    hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/org }
+}
+
+hook -group org-load-languages global WinSetOption filetype=org %{
+    hook -group org-load-languages window NormalIdle .* org-load-languages
+    hook -group org-load-languages window InsertIdle .* org-load-languages
+}
+
+provide-module orgmode %§
+
+# Faces
+# ‾‾‾‾‾
+
+set-face global org_todo          default,default+b@Error
+set-face global org_done          default,default+b@string
+set-face global org_priority      default,default@value
+set-face global org_heading       default,default+b@header
+set-face global org_section       default,default@function
+set-face global org_subsection    default,default@variable
+set-face global org_subsubsection default,default@module
+set-face global org_bold          default,default+b
+set-face global org_italic        default,default+i
+set-face global org_underline     default,default+u
+set-face global org_strikethrough default,default
```

Comment:
There are already global faces for markup (`header`, `section` etc), why not let the user override them whenever the filetype is `org`, instead of creating duplicate faces?

Reply:
Sorry, I didn't get what you meant. I've created these faces in order to let users override settings specifically for `org` if they want, without affecting other markup language highlighting. 

There's only `header` face for markup right now, and `heading` and `section` are different things in org by means of LaTeX

Reply:
Overloading faces for a specific filetype is trivial now that faces are scoped:

```
hook global WinSetOption filetype=org %{
    set-face window header red
}
```

Use global faces (i.e. the ones declared in colorschemes) as much as possible, users can overload them in their own configuration if needed: `bold`, `italic`, `header`…

I see you've already had a discussion about faces earlier with @mawww, maybe I'm just repeating what he/you said.